code/datums/autolathe/autolathe.dm

```diff
@@ -66,6 +67,7 @@ var/datum/category_collection/autolathe/autolathe_recipes
 	var/power_use = 0
 	var/is_stack
 	var/max_stack
+	var/no_scale
```

Comment:
If this is a 1-or-0 thing it should use TRUE/FALSE, rather than numbers. I'm not seeing anywhere that this specifically _isn't_ set to 1, but I'll mark them just in case.